pages/common/pprof.md

```diff
@@ -0,0 +1,24 @@
+# pprof
+
+> Command-line tool for visualization and analysis of profile data.
+> More information: <https://github.com/google/pprof>.
+
+- Generate a text report from `fibbo-profile.pb.gz` file, on fibbo binary:
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a text report from a specific file file, on fibbo binary:
```

What does the "on fibbo binary" bit here mean please? I'm a bit confused.